src/main/java/org/opentripplanner/model/plan/StreetLeg.java

```diff
@@ -75,6 +79,11 @@ public StreetLeg(
     this.elevationGained = calculateElevationGained(legElevation);
     this.elevationLost = calculateElevationLost(legElevation);
 
+    this.streetNotes = Set.copyOf(streetNotes);
+    this.pathwayId = pathwayId;
+    this.walkingBike = walkingBike;
+    this.rentedVehicle = rentedVehicle;
+    this.vehicleRentalNetwork = vehicleRentalNetwork;
     this.accessibilityScore = accessibilityScore;
   }
```

Comment:
I would pass in the builder as a single argument to the StreetLeg constructor:
 - It enforces the use of the builder
 - It is less code to maintain
 - the constructor can be package local scope

Reply:
Implemented in d8213497948e9142d6c44abb2159f639705bbdfb